app/merginapi.h

```diff
@@ -51,9 +51,9 @@ class MerginApi: public QObject {
      * when a response is received, parses project json, writes it to a cache text file and sets mMerginProjects.
      * Eventually emits listProjectsFinished on which ProjectPanel (qml component) updates content.
      * If listing has been successful, updates cached merginProjects list.
-     * @param
+     * @param withFilter If true, applies "input" tag in request.
      */
-    Q_INVOKABLE void listProjects();
+    Q_INVOKABLE void listProjects(bool withFilter = false);
```

Comment:
can we rename `withFilter` to something like `onlyWithInputTag` so that it's easier to understand what kind of filter is applied? Or even better, make it `QString filterTag = QString()` so that we can test with other tags in future...